jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/UnhandledServerUrlException.java

```diff
@@ -19,9 +19,9 @@
 import java.nio.file.Path;
 
 /** Thrown because the credential helper does not have credentials for the specified server URL. */
-public class NonexistentServerUrlDockerCredentialHelperException extends Exception {
+public class UnhandledServerUrlException extends CredentialRetrievalException {
 
-  NonexistentServerUrlDockerCredentialHelperException(
+  UnhandledServerUrlException(
```

Comment:
Hmm, maybe it might make more sense to prefix this with `DockerCredentialHelper` since it is specific to credential helpers, but it is quite a long name.

`DockerCredentialHelperUnhandledServerUrlException`

or maybe drop the `Docker`:

`CredentialHelperUnhandledServerUrlException`
`CredentialHelperNotFoundException`